dynawo/sources/Modeler/Common/DYNCompiler.cpp

```diff
@@ -388,6 +388,15 @@ Compiler::compileModelicaModelDescription(const shared_ptr<ModelDescription>& mo
     compileCommand += " --moFiles" + moFilesList + " --initFiles" + moFilesList;
   }
 
+  if (!additionalHeaderFiles_.empty()) {
+    string additionalHeaderList = "";
+    for (unsigned i = 0, iEnd = additionalHeaderFiles_.size(); i < iEnd; ++i) {
+      additionalHeaderList += " " + additionalHeaderFiles_[i];
+    }
+
+    compileCommand += " --additionalHeaderFiles" + additionalHeaderList;
```

Comment:
I will recommend using additionalHeaderList as the argument name when you are dealing with the string that represents a concatenation of the different additionalHeaderFiles to avoid confusion later on

Reply:
Fixed